### PR TITLE
@kanaabe: (Re)moved dependencies

### PR DESCRIPTION
**Pull request by craigspaeth**
Re-opening [this PR](https://github.com/artsy/force/pull/1211)

A little cleanup to remove some unused npm modules (found by [npm-check](https://github.com/dylang/npm-check)) and moving some tiny libs we put in the root of the project that are better suited as functions co-located with the app/component.

### CODE REVIEW

File: desktop/components/cookies/index.coffee [hunk 1]
```diff
@@ -1,5 +1,5 @@
 _ = require 'underscore'
-IS_TEST_ENV = require '../../lib/is_test_env.coffee'
+IS_TEST_ENV = require('sharify').data.NODE_ENV is 'test'
```
Comment (kanaabe) [inline desktop/components/cookies/index.coffee]: Simple :)

File: mobile/lib/analytics.coffee [hunk 2]
```diff
@@ -1,89 +0,0 @@
...
-  throw new Error('Requires modelName and id') unless modelName? and id?
-  "#{_s.capitalize(modelName)}:#{id}"
-
-module.exports.modelToLabel = (model) ->
-  throw new Error('Requires a backbone model') unless typeof model == 'object' and model?.constructor.name? and model?.get?('id')?
-  "#{_s.capitalize(model.constructor.name)}:#{model.get('id')}"
-
-module.exports.track_links = (args...) ->
-  unless typeof mixpanel is 'undefined'
-    mixpanel.track_links args...
```
Comment (kanaabe) [inline mobile/lib/analytics.coffee]: Does this file not get called from anywhere? Seems like setup code.

**Comment by kanaabe** [comment]
Nice spring cleaning! 🌷 💐

File: mobile/lib/analytics.coffee (hunk 2 above)
Comment (craigspaeth) [inline mobile/lib/analytics.coffee]: This module was our attempt at a home-grown Segment-like wrapper library. Now everything uses Segment and lives in the [analytics folder](https://github.com/artsy/force/tree/master/mobile/analytics). We cleaned this out of the desktop Force already, but didn't get around to doing it for MG. The only place I noticed that was left calling this library is [here](https://github.com/artsy/force/pull/1323/files#diff-5cb907221f9a8daef72180de61ebdafdL150) but I doubt we're using that event seeing as its using an old Mixpanel account ID and not funneling through Segment.

Comment (kanaabe) [inline mobile/lib/analytics.coffee]: 👍  Got it, thanks for the explanation! Good to go after 💚

File: circle.yml [hunk 3]
```diff
@@ -5,38 +5,21 @@ deployment:
...
   override:
+    - rm -rf node_modules/electron
+    - rm -rf node_modules/electron-download
+    - npm install electron
     - yarn install
 test:
   override:
     - "gem install danger --version '~> 4.0' && danger"
-    - case $CIRCLE_NODE_INDEX in 0) yarn mocha test/lib/* && yarn mocha $(find desktop/test -name '*.coffee') && yarn mocha $(find mobile/test -name '*.coffee') ;; 1) yarn mocha $(find desktop/components/*/test -name '*.coffee') && yarn mocha $(find desktop/components/**/*/test -name '*.coffee') ;; 2) yarn mocha $(find desktop/apps/*/test -name '*.coffee') && yarn mocha $(find desktop/apps/*/test -name '*.js') && yarn mocha $(find desktop/apps/*/**/*/test -name '*.coffee') ;; 3) yarn mocha $(find mobile/components/*/test -name '*.coffee') && yarn mocha $(find mobile/components/**/*/test -name '*.coffee') && yarn mocha $(find mobile/apps/*/test -name '*.coffee') && yarn mocha $(find mobile/apps/*/**/*/test -name '*.coffee'); esac:
+    - yarn test:
```
Comment (craigspaeth) [inline circle.yml]: I PR creeped this by cleaning up our circle.yml to always get a fresh install of Electron (hopefully circumventing the cache issue we've run into all the time). It also expands on scripts/test.sh so we can use that same script for Circle and local `yarn test`.

**Pull request merged**